### DISCUSSION TITLE
tailwind+next.js not working tried every documentation available onlinw

### DISCUSSION THREAD

**Discussion by ajinkyakotwal** [tailwindlabs/tailwindcss#10644] [Help]
1) index.js

import Head from 'next/head'
import Image from 'next/image'
import { Inter } from '@next/font/google'
import styles from '@/styles/Home.module.css'

const inter = Inter({ subsets: ['latin'] })

export default function Home() {
  return (
  <div className="bg-slate-500"> hello world </div>

  )
}

2) tailwind.config.js

module.exports = {
  content: [
    "./src/**/*.{js,ts,jsx,tsx}",
  ],
  theme: {
    extend: {},
  },
  plugins: [],
}

output:
hello world (in plain black color)

**Comment by wongjn** [comment]
Could I inquire as to what your expectations for the output would be please?

**Reply by ajinkyakotwal** [reply]
Just want make hello world in red text thats it. 
Tried that too.

**Reply by wongjn** [reply]
You'd need to use a class that applies the `color` property with a red value, like:
```html
<div class="text-red-600">hello world</div>
```

**Reply by ajinkyakotwal** [reply]
Did that on index.js but red color is not displaying on localhost also on this discussion their are not showing bg-color-500 which I did with hello word on below code
---------------------------------------------------------------
import Head from 'next/head'
import Image from 'next/image'
import { Inter } from '@next/font/google'
import styles from '@/styles/Home.module.css'
const inter = Inter({ subsets: ['latin'] })

export default function Home() {
  return (
  <div className="bg-slate-500"> hello world </div>

  )
}

-------------------------------------------------------------------------------------------------
below dependencies
-------------------------------------------------------------------------------
{
  "name": "next-frontend",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },

  "dependencies": {
    "@next/font": "13.1.6",
    "next": "13.1.6",
    "react": "18.2.0",
    "react-dom": "18.2.0"
  },

  "devDependencies": {
    "autoprefixer": "^10.4.13",
    "postcss": "^8.4.21",
    "tailwindcss": "^3.2.7"
  }
}
